### PR TITLE
Update sdk v3 links - Partially closes #1405

### LINKED ISSUE #1405: Insert specific content from SDK docs
### Description

- Split the current content of `docs-sdk` and move contents to the ROOT component
- Adjust links and flow of the content to the new structure

### PR DESCRIPTION
**Pull request by Tschakki**
Partially closes #1405

### CODE REVIEW

File: modules/ROOT/pages/guides/app-development/configuration.adoc [hunk 1]
```diff
@@ -3,15 +3,14 @@ Mona Bärenfänger <mona@lightcurve.io>
 :description: All configuration options, constants & the devnet genesis block which are relevant for configuring a blockchain application.
```
Comment (chris529) [inline modules/ROOT/pages/guides/app-development/configuration.adoc]:
```suggestion
:description: All configuration options, constants, & the devnet genesis block which are relevant for configuring a blockchain application.
```

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -3,7 +3,6 @@ Mona Bärenfänger <mona@lightcurve.io>
 :description: The Lisk SDK introduction describes its components, architecture, usage & how to get started.
```
Comment (chris529) [inline modules/ROOT/pages/index.adoc]:
```suggestion
:description: The Lisk SDK introduction describes its components, architecture, usage, and how to get started.
```

**Review by chris529 — COMMENTED** [review]
Looking good...just a couple of minor points 👍

File: modules/ROOT/pages/index.adoc (hunk 2 above)
Comment (Tschakki) [reply modules/ROOT/pages/index.adoc]:
```suggestion
```

**Review by shuse2 — APPROVED** [review]

**Pull request merged**